Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml

```diff
@@ -776,10 +776,15 @@ script:
       isArray: true
       name: dist_name
     - description: |-
-        A comma-separated list of IP addresses.
-        Example: 8.8.8.8,1.1.1.1.
+        A comma-separated list of private IP addresses.
```

Comment:
What effect will it have on customers already using the ip_list not as a private IP list.
I seems that you separated the behavior into 2 different arguments. (ip_list / public_ip_list)

Reply:
@omerKarkKatz , so my understanding (and discussing with XDR PS) is that the the ip_list argument pull the IP address of the endpoint, but it will always be a private IP.  This is even the case that there is a public IP address assigned
![image](https://github.com/demisto/content/assets/32227961/a9583fd7-9706-4e8d-9561-4fd81058d7fb)
![image](https://github.com/demisto/content/assets/32227961/3489e8a2-1aa7-4273-b701-1f106af6ee21)

Reply:
Thanks for the response, I didn't fully understand this but we will go over it in the Demo.